File: src/main/java/com/huythanh0x/training_thanhvh_java_spring_jwt_jpa/model/coupon/CourseJsonData.java
package com.huythanh0x.training_thanhvh_java_spring_jwt_jpa.model.coupon;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CourseJsonData {
    private String category;
    private String subCategory;
    private String courseTitle;
    private String level;
    private String author;
    private int contentLength;
    private float rating;
    private int numberReviews;
    private int students;
    private String language;
    private String headline;
    private String description;
}

File: modules/coupon-domain/src/main/kotlin/com/thanh0x/coursedeal/model/coupon/CourseJsonData.kt
package com.thanh0x.coursedeal.model.coupon

data class CourseJsonData(
    var category: String? = null,
    var subCategory: String? = null,
    var courseTitle: String? = null,
    var level: String? = null,
    var author: String? = null,
    var contentLength: Int = 0,
    var rating: Float = 0f,
    var numberReviews: Int = 0,
    var students: Int = 0,
    var language: String? = null,
    var headline: String? = null,
    var description: String? = null,
)
